packages/button/src/useRipple.tsx

```diff
@@ -11,7 +11,10 @@ export const useRipple: UseRipple = ({ onClick }) => {
   const [ripple, setRipple] = useState<React.ReactNode | null>(null)
 
   const handleClick = useCallback(
-    (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+    (
+      event: React.MouseEvent<HTMLAnchorElement, MouseEvent> &
```

Comment:
Why is it & here? It's probably anchor element | button element

Reply:
It works only with &